src/Components/Web/src/Virtualization/Virtualize.cs

```diff
@@ -171,7 +170,7 @@ protected override void BuildRenderTree(RenderTreeBuilder builder)
 
                 foreach (var item in itemsToShow)
                 {
-                    builder.AddContent(renderIndex, _itemTemplate, item);
+                    _itemTemplate(item)(builder);
```

Comment:
Does @SteveSandersonMS need to review this?

Reply:
This was his suggestion, he realized that adding/removing items could lead to poor behavior in the old implementation.